app/assets/stylesheets/layouts/_col_marginless.scss

```diff
@@ -0,0 +1,42 @@
+.l-2col-even--marginless {
```

Comment:
I think it could mean either one to be honest. As in, I wouldn't be shocked to find out it still had a gutter!